libs/clients/passports/src/lib/passports.config.ts

```diff
@@ -22,8 +22,10 @@ export const PassportsClientConfig = defineConfig<z.infer<typeof schema>>({
       fetch: {
         timeout: 10000,
         scope: [
-          NationalRegistryScope.passport,
           NationalRegistryScope.individuals,
+
+          // TODO: Add back in when ready.
```

Comment:
Is this todo needed before production?

Reply:
I have mentioned this to the service-providers a couple of times, this will not be available until later. 😞